gulp/sass.js

```diff
@@ -23,7 +23,7 @@ gulp.task('sass-extensions', function (done) {
 
 gulp.task('sass', function () {
   return gulp.src(config.paths.assets + '/sass/*.scss', { sourcemaps: true })
-    .pipe(sass({ outputStyle: 'expanded' }).on('error', function (error) {
+    .pipe(sass.sync({ outputStyle: 'expanded', logger: sass.compiler.Logger.silent }, false).on('error', function (error) {
```

Comment:
just checking what the `false` here is for? The other calls on this page don't have it

Reply:
I removed it in #1272 because as far as I can tell [sass.sync](https://github.com/dlmanning/gulp-sass/blob/master/index.js#L166-L169) does not accept a second argument.